yorkie/backend/sync/etcd/pubsub.go

```diff
@@ -41,11 +48,73 @@ func (c *Client) Publish(
 	publisherID *time.ActorID,
 	topic string,
 	event sync.DocEvent,
+) {
+	c.PublishToLocal(publisherID, topic, event)
+
+	c.memberMapMu.RLock()
+	defer c.memberMapMu.RUnlock()
+
+	for _, member := range c.Members() {
+		memberAddr := member.RPCAddr
+		if memberAddr != c.pubSub.AgentInfo.RPCAddr {
+			c.addBroadcastClient(memberAddr)
+		}
+
+		if memberAddr != c.pubSub.AgentInfo.RPCAddr {
+			c.publishToMember(
+				memberAddr,
+				publisherID,
+				topic,
+				event,
+			)
+		}
+	}
+}
+
+// PublishToLocal publishes the given event to the given Topic.
+func (c *Client) PublishToLocal(
+	publisherID *time.ActorID,
+	topic string,
+	event sync.DocEvent,
 ) {
 	c.pubSub.Publish(publisherID, topic, event)
+}
+
+// addBroadcastClient activates the broadcast grpc server and creates a client.
+func (c *Client) addBroadcastClient(addr string) {
+	if _, ok := c.broadcastClientMap[addr]; !ok {
+		conn, err := grpc.Dial(addr, grpc.WithInsecure())
+		if err != nil {
+			log.Logger.Error(err)
+		}
+
+		c.broadcastClientMap[addr] = &broadcastClientInfo{
+			client: api.NewBroadcastClient(conn),
+			conn:   conn,
+		}
+	}
+}
+
+// publishToMember publishes events to other agents.
+func (c *Client) publishToMember(
+	memberAddr string,
+	publisherID *time.ActorID,
+	topic string,
+	event sync.DocEvent,
+) {
+	bcm := c.broadcastClientMap[memberAddr]
```

Comment:
What does `bcm` stand for?

Reply:
It is a variable representing broadcastClientMap, but it seems to have been completely misnamed.
The name of the broadcastClientMap property has been changed, so the variable name has also been modified.